snappy/install.go

```diff
@@ -119,7 +119,9 @@ func doInstall(name, channel string, flags LegacyInstallFlags, meter progress.Me
 		return "", err
 	}
 
-	snap, err := mStore.Snap(name, channel, nil)
+	// devmode false preserves the old behaviour but we might want
+	// it to be set from flags instead.
+	snap, err := mStore.Snap(name, channel, false, nil)
```

Comment:
Hm, hm, I wonder if mStore.Snap() should take flags instead of a bool. from mStore.Snap(name, channel, false) it is not really clear what the false means here. One has to look at the signature of the function to find that out.